test/framework/toy_build.py

```diff
@@ -1040,6 +1040,41 @@ def test_minimal_toolchains(self):
         reprod_ec = os.path.join(self.test_installpath, 'software', 'toy', '0.0', 'easybuild', 'reprod', 'toy-0.0.eb')
         self.assertTrue(os.path.exists(reprod_ec))
 
+    def test_toy_toy(self):
+        """Test building two easyconfigs in a single go, with one depending on the other."""
```

Comment:
because the 1st install will result in a new module file, which is needed by the 2nd (but won't be in the cache unless we're careful), I wanted to make very sure we don't break that (very common) use case...